src/db.ts

```diff
@@ -1,23 +1,77 @@
 // db.ts
 import Dexie, { Table } from 'dexie'
 
-export interface Friend {
+export interface Student {
   id?: number
-  name: string
+  first_name: string
+  last_name: string
+  gender: 'male' | 'female' | 'nonbinary' | 'n/a'
```

Comment:
Would it be more appropriate to say "other" instead of "n/a"?

Reply:
Answered in [pull/1](https://github.com/phil208-ethics-project/phil208-ethics-project.github.io/pull/2)